apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -29,22 +29,21 @@
          counter_contract/1,
          dutch_auction_contract/1,
          environment_contract/1,
+         factorial_contract/1,
+         fundme_contract/1,
          identity_contract/1,
          maps_contract/1,
+         polymorphism_test_contract/1,
          simple_storage_contract/1,
          spend_test_contract/1,
+         stack_contract/1,
          null/1
         ]).
 
 -define(NODE, dev1).
 -define(DEFAULT_TESTS_COUNT, 5).
 -define(WS, aehttp_ws_test_utils).
```

Comment:
Unused? (Comment not in scope of this PR.)

Reply:
I know but it was there and easy to add so it just slipped in. And it is mentioned in the 2nd commit.